src/components/NotFoundPage/NotFoundPage.tsx

```diff
@@ -7,58 +7,39 @@ import Footer from 'components/Footer'
 import Navbar from 'components/Navbar'
 import LoadingPage from 'components/LoadingPage'
 
-import { Props, State } from './NotFoundPage.types'
+import { Props } from './NotFoundPage.types'
 import './NotFoundPage.css'
 
-const LOADING_TIMEOUT = 2000
+export default function NotFoundPage(props: Props) {
+  const [isLoading, setIsLoading] = useState<Boolean>(true);
```

Comment:
I think this should be typed `boolean` right? (not capitalized, meaning not the Constructor)

Reply:
Yep, my bad